omod/src/main/webapp/resources/htmlforms/mexico/consult.xml

```diff
@@ -191,6 +193,69 @@
         </p>
     </div>
 
+
+    <div class = "small-text section-container" style="color: #E94B3CFF;">
```

Comment:
Let's leave the text in its default color. Red means "the patient is about to catch fire"